Products/ZenHub/hubworker.zcml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<configure xmlns="http://namespaces.zope.org/zope">
+
+    <subscriber handler="Products.DataCollector.ApplyDataMaps.metrics.datamap_applied_time_tsdb_reporter" />
```

Comment:
The package path is `Products.DataCollector.ApplyDataMap.metrics...`
Note the singular `Map`.